guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,8 +4,49 @@
 [id="katello-headline-features"]
 == Headline Features
 
+=== Allow user to run Verify Content Checksum using Hammer, on container repositories
+Added option '--validate-contents true/false' to "hammer repository synchronize" command.
+
+=== Add ACS bulk refresh option, to refresh all ACS's
+Added option '--all' to "hammer alternate-content-source refresh" command.
+ACS bulk refresh through API silently sanitizes input ids.
+
+=== Add activation key details top bar
+When the setting 'Show Experimental Labs' is enabled, the Activation Key 
+details page will be replaced by the beginnings of a redesigned page.
+
+=== Content Views and CVV enhancements to filters and refresh
+Content View filter creation/edit button now says 'Save' instead of 'Edit Rule'.
+Content view dependency solving tracked for needs_publish.
+
+=== Pulpcore and plugins upgraded to Pulpcore 3.28
```

Comment:
I have a suggestion for some features to pull out that might interest users. Many of these features are things that Katello hasn't integrated with.

So, I would replace this whole section with something like:

```
=== Pulpcore and plugins updated to Pulpcore 3.28
Contains both fixes and features that Katello can use in the future to better manage content, such as:
https://github.com/pulp/pulp_rpm/issues/3135[More filter options on the packages API]
https://github.com/pulp/pulp_rpm/issues/3009[New setting for generated primary.xml timestamps to match upstream repository]
https://github.com/pulp/pulp_rpm/issues/2995[RPM replicator support]
https://github.com/pulp/pulpcore/issues/3538[New distribution hiding option]
https://github.com/pulp/pulpcore/issues/3835[OpenTelemetry support]
https://github.com/pulp/pulpcore/issues/3656[File size added to content application listing]
```